t/unit/backends/test_database.py

```diff
@@ -73,6 +82,8 @@ def test_table_schema_config(self):
             'task': 'foo',
             'group': 'bar',
         }
+        # disable table creation because databases foo and bar do not exist
+        self.app.conf.database_create_tables_at_setup = False
```

Comment:
Per my previous comment, this line will not be needed anymore

Reply:
oops, I forgot to remove it, will fix that too